autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -176,7 +176,8 @@
     protocol: TCP
     name: tcp2
   selector:
-    app: agent-node-{validator_ix}
+    app: agent-svc
```

Comment:
@Karrenbelt 
How about this one?

Reply:
I don't think it can get more generic than that

Reply:
How does this solve the problem where you have two autonomous services in the same cluster? They now are indistinguishable again?

Reply:
They are segmented by name space

Reply:
I.e. i have seperate name spaces for all of my agent services.